soc/nordic/nrf54l/CMakeLists.txt

```diff
@@ -10,3 +10,10 @@ zephyr_include_directories(.)
 if(CONFIG_ELV_GRTC_LFXO_ALLOWED)
   message(WARNING "WARNING! ELV mode feature is EXPERIMENTAL and may brick your device!")
 endif()
+
+# We need a buffer in memory in a static location which can be used by
+# the KMU peripheral. The KMU has a static destination address so we need
+# to make sure that this buffer is always at the start of RAM.
```

Comment:
Related PR talks about "top" of RAM. Should it state "top" here as well or is "start" a better term overall

Reply:
I replaced it with the specific address to make it more clear.